content/outcomes/application-development/pair-programming.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Pair Programming
```

Comment:
Consider changing the title to something more outcome oriented. Pair programming is not the outcome, but high quality code, "free" knowledge transfer, less time spent on code review, etc. are the outcomes.

Reply:
Interesting point. We'll reconsider all of the Modern Application Development articles/outcomes as a group to find a wholistic theme. The other topics have similar non-outcome titles.